src/transforms.rs

```diff
@@ -378,7 +379,7 @@ impl Transformer {
                 Ok(())
             }
             _ => Err(TransformError::ValueNotAnObject {
-                value: value.to_owned(),
+                value: value.message().to_owned(),
```

Comment:
`.into()` works here if implemented, saving us a clone

Reply:
I'm not sure what you're suggesting. `ValueNotAnObject` takes an owned value, I don't see how this code can into(U) its way out of that :smile:

Reply:
You're right. I was just doing the equivalent of find/replace without actually looking at the code. My suggestion would just make the borrow checker angry.